library/vyos_l3_interfaces.py

```diff
@@ -38,82 +37,84 @@
 
 DOCUMENTATION = """
 ---
-module: vyos_l3_interfaces
-version_added: 2.9
-short_description: Manages L3 interface attributes of VyOS network devices.
-description: This module manages the L3 interface attributes on VyOS network devices.
-author: Nilashish Chakraborty (@nilashishc)
+module:vyos_l3_interfaces
+version_added:2.9
+short_description:Manages L3 interface attributes of VyOS network devices.
+description:This module manages the L3 interface attributes on VyOS network devices.
+author:
+   - Nilashish Chakraborty (@NilashishC)
+   - Rohit Thakur (@rohitthakur2590)
 options:
   config:
-    description: The provided L3 interfaces configuration.
-    type: list
-    elements: dict
+    description:The provided L3 interfaces configuration.
```

Comment:
Any reason why we're removing the spaced between key and value in docs?

Reply:
Oh seems like I had a bug in my script to remove all those linting errors .